Source/Core/VideoBackends/OGL/OGLRender.cpp

```diff
@@ -751,6 +751,8 @@ Renderer::Renderer(std::unique_ptr<GLContext> main_gl_context, float backbuffer_
     OSD::AddMessage("This device's performance may be poor.", 60000);
   }
 
+  WARN_LOG_FMT(VIDEO, "Video Info: {}, {}, {}", g_ogl_config.gl_vendor, g_ogl_config.gl_renderer,
```

Comment:
Do we have something other than a warn?  Seems strange for it to be warning level

Reply:
We could change the value of `Verbosity` [here](https://github.com/dolphin-emu/fifoci/blob/master/runner/linux/Config-ogl/Logger.ini) (and in similar files for the other runners) to 4 for info.  Or we could use `NOTICE_LOG_FMT`, which puts it at the highest level (basically always visible) - since this info is apparently important enough to include via on-screen display, I don't think notice would be too out of place, but it does feel a bit overboard.

My main goal with this PR was just to get the version info so that I could do other testing.  (The info I wanted is [here](https://dolphin.ci/#/builders/16/builds/5846): `Video Info: Mesa/X.org, llvmpipe (LLVM 11.0.1, 256 bits), 4.5 (Core Profile) Mesa 20.3.5` - unfortunately we can't travel back in time, so I can't know what it was a week or so ago before some recent issues started appearing.)

Reply:
Just to clarify that: I'd be fine with changing the verbosity in a FifoCI PR and changing the log level back to warn; the switch to warn was so that I could quickly get the information I wanted to do other debugging.  The two commits should be squashed together at least.

Reply:
I've switched it to warning level, which will work when combined with dolphin-emu/fifoci#47.